randovania/games/am2r/exporter/joke_hints.py

```diff
@@ -8,6 +8,7 @@
     "*Adam has not yet authorized the use of this hint.*",
     "*Did you know that Frog Legs are delicious with some Goat Butter, Garlic and Parsley?*",
     "*If you see an Omega Metroid shinesparking towards your general direction, run.*",
+    "*For the hint to Return... we must get rid of Proteus Ridley, find the Metroid DNA and repair the Chozo Seals...*",
```

Comment:
```suggestion
    "*For the hint to Return... we must get rid of Proteus Ridley, find the Metroid DNA, and repair the Chozo Seals...*",
```

Reply:
If I do this, then the line gets 1 character over the limit :'(

Reply:
Instead of `get rid of`, could do eradicate, remove, eliminate, defeat, destroy

Reply:
`get rid of` is part of the original quote.
> For peace to return... we must get rid of Smithy, find the Star Pieces, and repair the Star Road...

Reply:
Oh

Reply:
Could. I thought it'd be better to be completely on the nose just to make sure people don't think there's some ridley fight somewhere.

Reply:
For hints to Return?